requirements.in

```
@@ -17,7 +17,7 @@ invoke
 importlib-metadata
 jenkins-job-builder
 jinja2
-juju
+juju <= 3.0
```

i see somehow `requirements.txt` ok, but shouldn't this be this:

```suggestion
juju<3.0
```